tools/phasestack.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import sys
+from getopt import gnu_getopt, GetoptError
+from collections import namedtuple
+
+from libgcopdd import Event, UserError, open_dump_entry
+
+PhaseStack = namedtuple('PhaseStack', 'size, data')
```

Comment:
Same as in DepMat, wouldn'ŧ it be better if this would be a class with methods on it? For example you can'ŧ use depmat's aggregate and phasestack's aggregate at once without aliasing one of them. That seems unnecessary.